### ISSUE TITLE
[Documentation] Beta TestNet Preparation & G2M

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/poktroll#901]
## Objective

Document & Track everything that needs to be done up, during and after Beta TestNet.

## Origin Document

https://www.notion.so/buildwithgrove/Beta-TestNet-Prep-G2M-7af6ee47925c4087b6ae11a13874fef2?pvs=4

## Goals

- Identify all missing gaps that need to be completed PRIOR TO Beta TestNet launch
- Communicate & align with the community + foundation on where things are
- Incentivize new users to onboard to Shannon Beta TestNet
- Start preparing post Beta TestNet work + path to MainNet

## Deliverables

- [x] A Beta TestNet Launch Document w/ all the outstanding items
- [x] A blog post announcing Beta TestNet and next steps

## Non-goals / Non-deliverables

- Any hands-on tactical work/changes
- Concrete details of Beta TestNet incentives

---

**Creator**: @Olshansk 
**Co-Owners**: @fredteumer

**Comment by Olshansk** [comment]
Published: https://medium.com/decentralized-infrastructure/an-update-from-grove-on-shannon-beta-testnet-path-the-past-the-future-5bf7ec2a9acf
